src/db.rs

```diff
@@ -434,13 +429,14 @@ impl Database {
     /// Returns a [`WriteTransaction`] which may be used to read/write to the database. Only a single
     /// write may be in progress at a time
     pub fn begin_write(&self) -> Result<WriteTransaction> {
```

Comment:
Can you update the docs to say that it will block if a write is in progress?

Reply:
Done.